src/libcrun/linux.c

```diff
@@ -3514,7 +3518,8 @@ libcrun_run_linux_container (libcrun_container_t *container, container_entrypoin
     {
       __attribute__ ((unused)) cleanup_pid pid_t pid_to_clean = pid;
 
-      ret = libcrun_save_external_descriptors (container, pid, err);
+      /* this is safe to do because the std stream files were not changed since the clone().  */
+      ret = libcrun_save_external_descriptors (container, 0, err);
```

Comment:
not a blocker and nit or more like a stupid question: Should we replace `0` to `-1` for `self` although not applicable in this case but `0` might give people impression this this `swapper` or `sched`.

Reply:
wouldn't that give the impression that `0` is a valid PID? :D

Reply:
my bad yes :sweat_smile: .